src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -473,16 +477,18 @@ std::wstring CascadiaSettings::ExpandEnvironmentVariableString(std::wstring_view
 // - name: the name of the new profile.
 // Return Value:
 // - A Profile, ready to be filled in
-Profile CascadiaSettings::_CreateDefaultProfile(const std::wstring_view& name)
+Profile CascadiaSettings::_CreateDefaultProfile(std::wstring_view name)
 {
-    Profile newProfile{
-        Microsoft::Console::Utils::CreateV5Uuid(
-	    TERMINAL_PROFILE_NAMESPACE_GUID,
-	    gsl::as_bytes(gsl::make_span(name))
-	)
-    };
+    auto profileGuid{ Utils::CreateV5Uuid(TERMINAL_PROFILE_NAMESPACE_GUID, gsl::as_bytes(gsl::make_span(name))) };
+    Profile newProfile{ profileGuid };
 
     newProfile.SetName(static_cast<std::wstring>(name));
 
+    std::wstring iconPath{ PACKAGED_PROFILE_ICON_PATH };

```

Comment:
would `std::filesystem` be useful here?

Reply:
there's no formal `url` type, and this is a url... i didn't want to risk it :)